src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/GenericComponent.tsx

```diff
@@ -76,13 +78,10 @@ export function GenericComponent(props: IGenericComponentProps) {
 
     const dataModelField = props.dataModelBindings[key];
     FormDataActions.updateFormData(dataModelField, value, props.id);
-    // Disable single field validation, enable when supported server-side
-    // const component = layoutElement as ILayoutComponent;
-    // if (component && component.triggerValidation) {
-    //   const { org, app, instanceId } = window as Window as IAltinnWindow;
-    //   const url = `${window.location.origin}/${org}/${app}/instances/${instanceId}`;
-    //   ValidationActions.runSingleFieldValidation(url, dataModelField);
-    // }
+    if (props.triggers && props.triggers.includes("validation")) {
```

Comment:
Would suggest to add this as an enum in `types/index.ts`, where we could add more triggers in the future. See f.ex. `ProcessSteps` enum as example.

Reply:
Added type Triggers